File: booking-service/src/main/java/com/javasolution/app/bookingservice/responses/Slot.java
package com.javasolution.app.bookingservice.responses;

import lombok.Data;

import java.time.LocalDateTime;

@Data
public class Slot {

    private Long id;
    private LocalDateTime startTime;
    private LocalDateTime finishTime;
    private Boolean isBooked;
    private Long resourceId;
}

File: front-app/src/app/models/Slot.ts
import { NumberValueAccessor } from '@angular/forms';

export class Slot {
  id: number;
  startDate?: string;
  startTime: string;
  finishDate?: string;
  finishTime: string;
  divisor?: number;
  isBooked: string;
  resourceId: number;
}
